util/llpcPipelineDumper.cpp

```diff
@@ -640,6 +640,9 @@ void PipelineDumper::DumpPipelineShaderInfo(
 #if LLPC_CLIENT_INTERFACE_MAJOR_VERSION >= 35
     dumpFile << "options.disableLicm = " << pShaderInfo->options.disableLicm << "\n";
 #endif
+#if LLPC_CLIENT_INTERFACE_MAJOR_VERSION >= 39
+    dumpFile << "options.unrollThreshold = " << pShaderInfo->options.unrollThreshold << "\n";
```

Comment:
Please check if you update the hash in UpdateHashForPipelineShaderInfo()

Reply:
Good spot - I'd missed that.